### DISCUSSION TITLE
2 aesthetic elements from Notion

### DISCUSSION THREAD

**Discussion by nothing2obvi** [docmost/docmost#41] [Ideas]
Hello! First I want to say that Docmost looks awesome and could potentially (finally) be what I switch to from Notion. I just have two ideas/questions about elements from Notion that I would love to see incorporated:

1.) The page icon (emoji) also being on the actual page, next to or above the page title, instead of only in the sidebar

2.) The Unsplash cover photos that Notion has for each page (or at least something similar). They really set the mood of the page for me and help me stay organized in my ADHD brain. I understand it's not important functionally but it's weirdly almost a must-have for me.

Any thoughts? Thanks!

**Comment by Philipinho** [comment]
Thank you for your input.

1. The page Icon should be an easy fix.
2. We have a database column for `cover_photo`. I am yet to work on its  implementation. Good to know your view on its impact. It will definitely come.

**Reply by nothing2obvi** [reply]
Thanks for your reply. These are both great to hear. Looking forward to them!
